syntax = "proto3";

package test.simple;

message SimpleMessage {
    string name = 1;
    int32 id = 2;
    repeated string tags = 3;
}

message TestRequest {
    string client_info = 1;
    uint32 version_major = 2;
    uint32 version_minor = 3;
}

message TestResponse {
    bool success = 1;
    string message = 2;
    int32 code = 3;
}